airflow/dags/deploy_dbt_docs/METADATA.yml

```diff
@@ -0,0 +1,19 @@
+description: "Builds dbt artifacts and syncs documentation to Metabase."
```

Comment:
what does "builds dbt artifacts" mean in this case? not generating new manifest, right?

Reply:
Technically it does have to compile the project, and it's doing that from the project source rather than what's in the latest artifacts GCS bucket.

Reply:
This makes me a tad nervous re: things getting out of sync, but I guess it's fine?